Xamarin.Forms.Build.Tasks/NodeILExtensions.cs

```diff
@@ -161,17 +161,17 @@ public static bool CanConvertValue(this ValueNode node, ILContext context, TypeR
 			if (targetTypeRef.ResolveCached().BaseType != null && targetTypeRef.ResolveCached().BaseType.FullName == "System.Enum")
 				yield return PushParsedEnum(targetTypeRef, str, node);
 			else if (targetTypeRef.FullName == "System.Char")
-				yield return Instruction.Create(OpCodes.Ldc_I4, Char.Parse(str));
+				yield return Instruction.Create(OpCodes.Ldc_I4, (int)Char.Parse(str));
```

Comment:
those casts can be `unchecked`

Reply:
You mean to wrap them with `unchecked`? That is right, but I wonder if it makes difference.

Reply:
you're right, it doesn't change a thing. my wrong.